backend/routes.py

```
@@ -35,6 +37,96 @@ def register_event_handlers():
+                "language": repo_data.get("language"),
+                "created_at": repo_data.get("created_at"),
+                "updated_at": repo_data.get("updated_at"),
+                "url": repo_data.get("html_url"),
+                "owner": {
+                    "login": repo_data.get("owner", {}).get("login"),
+                    "avatar_url": repo_data.get("owner", {}).get("avatar_url"),
+                }
+            }
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)